src/common.ts

```diff
@@ -20,16 +20,21 @@ export class GaxiosError<T = any> extends Error {
   code?: string;
   response?: GaxiosResponse<T>;
   config: GaxiosOptions;
+  status: Number;
```

Comment:
Doc + type change. There are a few differences between [`number` and `Number`](https://stackoverflow.com/questions/67155108/what-is-the-difference-between-number-and-number-in-typescript)

```suggestion
  /**
  * An HTTP Status code.
  * See {@link https://developer.mozilla.org/en-US/docs/Web/API/Response/status Response: status property}
  *
  * @example
  * 500
  */
  status: number;
```